semigroupoids.cabal

```diff
@@ -176,12 +176,12 @@ library
   hs-source-dirs: src
 
   exposed-modules:
-    Data.Bifunctor.Apply
-    Data.Functor.Alt
-    Data.Functor.Apply
-    Data.Functor.Bind
-    Data.Functor.Bind.Class
-    Data.Functor.Bind.Trans
+    Data.Bifunctor.Semiapplicative
+    Data.Functor.Semialternative
+    Data.Functor.Semiapplicative
+    Data.Functor.Semimonad
```

Comment:
Or semigroupad? My google finds `Semimonad` which is apparently different things, but virtually nothing on `Semigroupad`